personal-practice/car.java

```diff
@@ -0,0 +1,37 @@
+import java.awt.*;
+import java.awt.Color;
+
+public class car {
+
+    // Data Types: 
+    // int
+    // double (float)
+    // string
+    // Color (from awt library)
+    // boolean
+
+    double averageMilesPerGallon;
+    String licensePlate;
+    Color paintColor;
+    boolean areTaillightsWorking;
+
+    public car(double inputAverageMPG, 
+    String inputLicensePlate, 
+    Color inputPaintColor, 
+    boolean inputAreTaillightsWorking) {
```

Comment:
Wrapped line is indented [by 4 spaces](https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent).